pr-data.csv

```diff
@@ -2809,7 +2809,7 @@ https://github.com/apache/shenyu,10375c5204c67f9889235f19fe2d6e8eb0438a5a,shenyu
 https://github.com/apache/shenyu,10375c5204c67f9889235f19fe2d6e8eb0438a5a,shenyu-client/shenyu-client-http/shenyu-client-springmvc,org.apache.shenyu.client.springmvc.init.SpringMvcClientEventListenerTest.testShenyuBeanProcess,OD,Opened,https://github.com/apache/shenyu/pull/4252,
 https://github.com/apache/shenyu,10375c5204c67f9889235f19fe2d6e8eb0438a5a,shenyu-client/shenyu-client-http/shenyu-client-springmvc,org.apache.shenyu.client.springmvc.init.SpringMvcClientEventListenerTest.testWithShenyuClientAnnotation,OD,Opened,https://github.com/apache/shenyu/pull/4252,
 https://github.com/apache/shenyu,af0444f5d6efabedc565b29f7a7384b4e79f3fc7,shenyu-loadbalancer,org.apache.shenyu.loadbalancer.cache.UpstreamCacheManagerTest.findUpstreamListBySelectorIdTest,OD,,,
-https://github.com/apache/shiro,f14a7d9dd429439b6fdecc596af0235ec1126414,support/cdi,org.apache.shiro.cdi.AnnotatedTypeWrapperTest.removeAnnotations,ID,,,
+https://github.com/apache/shiro,f14a7d9dd429439b6fdecc596af0235ec1126414,support/cdi,org.apache.shiro.cdi.AnnotatedTypeWrapperTest.removeAnnotations,ID,Opened,https://github.com/apache/shiro/pull/1149/,
```

Comment:
Please remove '/' from the end of the URL (and ideally improve the checker to check for that).